CMakeLists.txt

```diff
@@ -50,12 +78,12 @@ add_executable(rt-tests
   tests/test_service_quality.cpp
 )
 
-add_dependencies(rt-tests googletest)
+add_dependencies(rt-tests googletest roc)
 
 find_package(Threads)
 
 target_link_libraries(rt-tests
-  roc
+  -l:libroc.a
```

Comment:
Wouldn't this fail when DOWNLOAD_ROC is OFF?

Reply:
Fixed